<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>AgNO3</author>
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>2560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>伤害计算器 v1.3</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="autoSave">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadBtn">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Authur">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Contact：米游社 AgNO3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="Main">
     <item>
      <layout class="QVBoxLayout" name="Input">
       <item>
        <layout class="QHBoxLayout" name="Atk">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>攻击力</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="atk">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>*</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Talent">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>技能倍率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="talent">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="dmg_buff">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>伤害提高</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dmgBuff">
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Crit_rate">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>暴击率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="crit_rate">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Crit_DMG">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>暴击伤害</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="crit_dmg">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>50</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Buff">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>伤害加成</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="buff">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Resistance">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>怪物抗性</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="resistance">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="React">
         <item>
          <widget class="QRadioButton" name="nora">
           <property name="text">
            <string>无倍率反应</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ra150">
           <property name="text">
            <string>1.5倍率反应</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ra200">
           <property name="text">
            <string>2倍率反应</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Element_master">
         <item>
          <widget class="QRadioButton" name="EM_btn">
           <property name="text">
            <string>元素精通</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Elem_mas">
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="EMbuff_btn">
           <property name="text">
            <string>反应伤害提升</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="em_buff">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="reactBuff_">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>系数加成</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reactBuff">
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Level">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>角色等级</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="char_level">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>90</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>怪物等级</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="mons_level">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>减防御力</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="defense">
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Output">
       <item>
        <widget class="QPushButton" name="cal">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>120</height>
          </size>
         </property>
         <property name="text">
          <string>计算</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="res">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="placeholderText">
          <string>计算结果</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>atk</tabstop>
  <tabstop>crit_rate</tabstop>
  <tabstop>crit_dmg</tabstop>
  <tabstop>buff</tabstop>
  <tabstop>talent</tabstop>
  <tabstop>resistance</tabstop>
  <tabstop>nora</tabstop>
  <tabstop>ra150</tabstop>
  <tabstop>ra200</tabstop>
  <tabstop>Elem_mas</tabstop>
  <tabstop>char_level</tabstop>
  <tabstop>mons_level</tabstop>
  <tabstop>cal</tabstop>
  <tabstop>res</tabstop>
 </tabstops>
 <includes>
  <include location="local">Genshin Damage Calculator v1.2</include>
  <include location="local">Made by AgNO3</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>EMbuff_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>em_buff</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EM_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Elem_mas</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
